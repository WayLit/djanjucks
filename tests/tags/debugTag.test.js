import djanjucks from '../../src'

describe('debug tag', () => {
  it('outputs the context', () => {
    const result = djanjucks.renderString('{% debug %}', {
      items: ['A', 'B', 'C'],
      date: new Date('01-01-2019'),
      object: {
        key: 'value'
      },
      number: 100,
      bool: true
    })

    expect(result).toMatchSnapshot()
  })
})
