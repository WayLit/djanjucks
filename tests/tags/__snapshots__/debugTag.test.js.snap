// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`debug tag outputs the context 1`] = `
"{
  &quot;items&quot;: [
    &quot;A&quot;,
    &quot;B&quot;,
    &quot;C&quot;
  ],
  &quot;date&quot;: &quot;2019-01-01T00:00:00.000Z&quot;,
  &quot;object&quot;: {
    &quot;key&quot;: &quot;value&quot;
  },
  &quot;number&quot;: 100,
  &quot;bool&quot;: true
}"
`;
